benchmark/snn_llm/_spikegpt_author.py

```
@@ -0,0 +1,149 @@
+        step_mode="m",
+        backend=backend,
+        store_v_seq=True,
+    )
+
+
+def _git_head(root: Path) -> str | None:
+    if not (root / ".git").exists():
+        return None
+    try:
+        completed = subprocess.run(
+            ["git", "rev-parse", "HEAD"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)